sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -41,21 +41,21 @@ enum {
 	SOF_I2S_SSP5 = BIT(5),
 };
 
-#define SOF_RT711_JDSRC(quirk)		((quirk) & GENMASK(1, 0))
-#define SOF_SDW_FOUR_SPK		BIT(2)
-#define SOF_SDW_TGL_HDMI		BIT(3)
-#define SOF_SDW_PCH_DMIC		BIT(4)
-#define SOF_SSP_PORT(x)		(((x) & GENMASK(5, 0)) << 5)
-#define SOF_SSP_GET_PORT(quirk)	(((quirk) >> 5) & GENMASK(5, 0))
-#define SOF_RT715_DAI_ID_FIX		BIT(11)
-#define SOF_SDW_NO_AGGREGATION		BIT(12)
+#define SOF_RT711_JDSRC(quirk)		((quirk) & GENMASK(3, 0))
```

Comment:
@bardliao can we move variable definition to the end so we don't need to change other definitions ?

Reply:
> @bardliao can we move variable definition to the end so we don't need to change other definitions ?

@RanderWang I thought about it, but it is too crazy to have more than 15 JD modes in a codec. So I think 4 bits for JD mode is more than enough.